_posts/2021-04-21-decoding-from-memory-bug.md

```diff
@@ -10,7 +10,7 @@ category: Security Alerts
 
 On April 5th, 2021, a bug in the Solidity ABI decoder v2 was reported by
 John Toman of the Certora development team. Certora's bug disclosure post
-can be found here: [Memory Isolation Violation in Deserialization Code](https://www.certora.com/blog/deserialization.html).
+can be found here: [Memory Isolation Violation in Deserialization Code](https://medium.com/certora/memory-isolation-violation-in-deserialization-code-certora-bug-disclosure-aece7cd00562).
```

Comment:
Could also just point to an archive.org link 🤓

Reply:
Or I can put the archive link into a footnote like: "The original blog post was published [here](http://web.archive.org/web/20210409180710/https://www.certora.com/blog/deserialization.html)." Do you support footnotes via GitHub markdowns?

E.g. Hello[^1]

[^1]: This is a footnote.

Raw version:

```bash
E.g. Hello[^1]

[^1]: This is a footnote.
```

Reply:
I'm not a decision maker on this, but my personal opinion is that updating blog posts (i.e. things which are a snapshot of time) is not good, as the updates may distort the meaning.

Yes, the original link could have been updated any time anyhow For this reason my preference would be probably the footnote option (or any other marking to state what the original URL was via archive.org).

It's up to @ekpyron however.

Reply:
Resolving the comment since the PR got merged.